ui/src/hooks/useSearchByKeywordData.js

```diff
@@ -16,12 +16,15 @@ function useSearchByKeywordData(query) {
   }
 
   useEffect(() => {
+    // eslint-disable-next-line no-alert
+    const showAlert = () => window.alert('Error occurred while downloading jobs!');
     async function fetchData() {
       if (isDownloaded !== true && isDownloading !== true) {
         setDownloading(true);
         dispatch(clearJobsAction());
         if (lastQuery.length > 0) {
-          const jobs = await findJobsByKeyword(lastQuery);
+          const jobs = await findJobsByKeyword(lastQuery)
+            .catch(() => showAlert());
```

Comment:
- [ ] could we use `.catch(showAlert);`?

Reply:
After moving `window.alert()` to separate file we now have:
```
.catch(() => showAlert('Bla bla bla...'));
```